server/jetstream_cluster_4_test.go

```diff
@@ -1371,19 +1537,19 @@ func TestJetStreamClusterStreamOrphanMsgsAndReplicasDrifting(t *testing.T) {
 	// Clustered file based with discard old policy and no limits.
 	t.Run("R3F_DO_NOLIMIT", func(t *testing.T) {
 		params := &testParams{
-			restartAny:       false,
-			ldmRestart:       true,
-			rolloutRestart:   true,
-			restarts:         3,
-			checkHealthz:     true,
-			reconnectRoutes:  true,
-			reconnectClients: true,
+			restartsMode:      ClusterRollout,
+			restarts:          3,
+			ldmRestart:        true,
+			checkHealthz:      true,
+			disconnectRoutes:  true,
+			disconnectClients: true,
 		}
 		test(t, params, &nats.StreamConfig{
 			Name:       "OWQTEST_R3F_DO_NOLIMIT",
 			Subjects:   []string{"MSGS.>"},
 			Replicas:   3,
 			Duplicates: 30 * time.Second,
+			Retention:  nats.WorkQueuePolicy,
```

Comment:
this one not meant to be a workqueue

Reply:
What should it be @wallyqs ?

Reply:
should be LimitsPolicy, added to the PR

Reply:
Should the name be updated? it contains `WQ` and `NO_LIMITS`.
The test logic also need to change, we can't expect `stream.messages` to match `SUM(consumer.pending)`

Reply:
it is a limits based stream without limits, I would remove the WQ from the stream name which I left in... but it is covering a particular failure mode, you can rename it into something like `TEST_R3F_DO_LIMITS` (just limits based with discard old)

Reply:
Ok, moving this PR back to draft until i address it.